docs/pages/e-mail-signup-forms.md

```diff
@@ -49,6 +116,42 @@ variation_groups:
         variation_specs: |-
           * Heading: H3
           * Body copy: Avenir paragraph
+        variation_code_snippet_rendered: >-
+          <div class="o-email-signup__inset" style="max-width: 370px">
```

Comment:
Looks like the inset is still here. This will render the markup without the styles.